participants/sector-checker.md

```diff
@@ -0,0 +1,51 @@
+# sector-checker
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+- project's name : sector_checker
+- slack handles: https://filecoinproject.slack.com/team/U01480QEWQ5
+
+## (1) Brief Project Description
+This is a tools to ensure the sectors pass the window PoST
+
+## (2) Link to Project UI
+
+https://github.com/irocn/sector-checker
+
+## (3) What does your application/UI do?
+- Check the sectors health status
+- Find the sanity or faulty sectors before window PoST
+- Analysis the sector log to improve seal efficiency and find the potential problem
+
+## (4) If your project is using a curated dataset, which one are you using?
+No curated dataset
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+The binary file itself will be stored. And the json log files
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+1G or more, people could save the logs on the filecoin, the more the user, the more the logs
```

Comment:
The filecoin logging infomration or chain data is an interesting approach.  You may want to consider https://proofs.filecoin.io/ or https://trusted-setup.filecoin.io/ as additional data that can increase your data volume and provide a valuable service.  Both of these (and other blockchain examples) are parr of the curated dataset.